shell/packages/accounts-email-token/token_templates.html

```diff
@@ -0,0 +1,3 @@
+<template name="_emailTokenError">
```

Comment:
This template ends up looking pretty weird. The top bar is missing, except for the sign-in link in the upper-right. Not sure if there's much we can do about it while keeping this package independent, though...